### PR TITLE
Edits to in-product text (titles)

### PR DESCRIPTION
**Pull request by AninditaBasu**
#### Summary

Edits to titles of windows, dialog boxes, etc, so that the [UI Text Guidelines](https://docs.mattermost.com/process/documentation-UItext-guidelines.html) are met.

This is part of GSoD'19 work.

### CODE REVIEW

**Comment by adamjclarkson** [comment]
Thanks @AninditaBasu

**Comment by wiersgallak** [comment]
@esethna adding you for visibility.  Adam, Justine and I have all given this a pass.  Adding QA as part of our normal PR flow for web-app tickets. 

@AninditaBasu Can you perform the sync & merge to help clear the checks?

**Comment by AninditaBasu** [comment]
Hi @wiersgallak, I resolved the conflicts, so I guess this PR is ready to be merged now?

**Comment by esethna** [comment]
Looks good! QA may want to have a look at this before it's merged

File: i18n/en.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
   "activity_log.os": "OS: {os}",
   "activity_log.sessionId": "Session ID: {id}",
   "activity_log.sessionsDescription": "Sessions are created when you log in to a new browser on a device. Sessions let you use Mattermost without having to log in again for a time period specified by the System Admin. If you want to log out sooner, use the 'Logout' button below to end a session.",
-  "add_channels_to_scheme.title": "Add Channels To **Channel Selection** List",
+  "add_channels_to_scheme.title": "Add Channels to **Channel Selection** List",
```
Comment (saturninoabril) [inline i18n/en.json]: This requires change to defaultMessage in corresponding components where the translation is applied.

Comment (AninditaBasu) [reply i18n/en.json]: @saturninoabril I'll open a separate PR for default messages in all of the components, if that's okay? For this PR, I was hoping to confine the changes to only one file (and one group of the strings, namely Titles)

Comment (saturninoabril) [reply i18n/en.json]: I'm 3/5 that it should be part of this PR.  Separating it will be hard to back track in the future.

Comment (AninditaBasu) [reply i18n/en.json]: Hi @saturninoabril, I edited the defaultMessage in the corresponding components for all of these titles. Please have a look?

**Comment by saturninoabril** [comment]
@AninditaBasu Tests are failing; need to update snapshot. I'll push commit to fix it then will merge.

Do you have corresponding Jira ticket or Github issue number?

**Comment by saturninoabril** [comment]
@AninditaBasu Hmm, I'm getting error pulling your branch since you're on `master`.  You may update the test snapshots by `npm run updatesnapshot` or `npm run test -- -u`.

**Comment by AninditaBasu** [comment]
@saturninoabril 
I dont have `npm` installed. Are the tests failing because I updated the files under the `_snapshots` folders of the components? If so, I can revert the changes to the `_snapshots` files.
Am sorry for all the trouble.

**Comment by saturninoabril** [comment]
No worries. What you did on snapshots are correct.  There's only other snapshots and unit test that needed to update as well.  Should be good now - https://github.com/mattermost/mattermost-webapp/pull/4649/commits/80e66c064274d8f8f693d27a589f57630f86ec90

I've seen similar tickets but can't tell which one this is for.
- https://mattermost.atlassian.net/browse/MM-15686
- https://mattermost.atlassian.net/browse/MM-15687
- https://mattermost.atlassian.net/browse/MM-15688

I'll defer to @wiersgallak about Jira (on linking or creation).  I believed we aim to have Jira ticket for every merged PR but not sure if there's an exception.

**Pull request merged**